### ISSUE TITLE
[Programmatic Access - Bot Framework Web chat - Carousel]: Headings are not proper on the page. Multiple h1's are defined on carousel response article.

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4726]
### **User Experience:**
Screen reader users gets affected as they cannot understand the page structure if multiple h1 heading are defined on the page. Screen reader users cannot understand the heading hierarchy and cannot navigate effectively.
### **Note:**
User credentials should NOT be included in the bug.
### **Repro Steps:**

- Launch URL: https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/ on edge dev.
- Web chat page opens.
- Press 4-tab keys to reach 'Type your message' field. Enter the "Carousel" keyword on it. Press 1-tab key to reach the 'send' button and activate it with the enter key.
- Press 2-shift tabs and 1-down arrow to reach the received card. Activate it with the enter key. Focus lands on 'mark it down' link. Turn on AI-headings.
- Verify whether the headings are properly defined or not.
### Actual Result:
Headings are not proper on the page. Multiple h1's are defined on carousel response article.
**Observation:**
There is no main heading. H1 is defined for each tile/ card of the carousel response article.
### **Expected Result:**
Headings hierarchy should be proper on the page. Multiple h1's should not be defined on carousel response article.
H1 should be defined for main content of page. Tile/ card headings should be defined as h2's.
### **MAS Reference:_**
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

### **Reference Links:**

- [Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [Link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDV8MDF8fDNkZjhhMWU0NTA0YTQwM2MyNGRhMDhkYWY1NTkwMDNlfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgwOTIwNTk4MzEzMTM2MjZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkzTXpZd09URTRNall3TXpzeE5qY3pOakE1TVRneU5qQXpPekU1T2pjMllUWXlNRE5pTnpZeFpqUmlObVE0T1dKbE0yRm1aVFZpTjJZd1pHSmhRSFJvY21WaFpDNTJNZz09fDBiZDAxMWU1M2M0YjQ0NGYyNGRhMDhkYWY1NTkwMDNlfGI0NDRlYzhjODQ4YzRiODI4Mzk1NmRjNDdlZjQ3NDky&sdata=dlJVM3lpMWQrRmtpc09HVTI0REh4NlVrU00yMG5xd0JlUFdBOHZuaHNOVT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-pgopinadhu%40microsoft.com&OR=Teams-HL&CT=1673616420039&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzAxMDUwOTQwMCIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D). “HCL Staff should not log any third-party external bugs. Should be routed to edad team”.
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.
### **Test Environment:**

OS: Windows 11 Enterprise 22H2 (OS build: 22621.1555)
Browser: Edge Version 114.0.1807.6 (Official build) dev (64-bit)
URL: [Web Chat: Full-featured bundle (microsoft.github.io)](https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/)
Tools: Developer Tool, Accessibility Insights for Web
Matrix: Edge dev+ Narrator.

**Comment by pgopinadhu** [comment]
#A11yMAS;#A11ySev3;#MAS1.3.1;#AILimited;#HCL;#Accessibility;#Benchmark;#Win11-Edge(Chromium);

**Comment by compulim** [comment]
Particularly.

![image](https://github.com/microsoft/BotFramework-WebChat/assets/1622400/69c9b4a5-fb52-4976-949a-e5e6455b56eb)

In our code, we convert the card to AC and there seems to be no way to modify `aria-level` unless we hack it again.

<img width="454" alt="image" src="https://github.com/microsoft/BotFramework-WebChat/assets/1622400/efc79917-37e2-4994-8483-d4ebcfb79304">

**Comment by compulim** [comment]
Fixed by #4747.
